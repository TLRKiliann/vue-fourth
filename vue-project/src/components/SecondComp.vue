<script lang="ts">
import ThirdComp from './ThirdComp.vue';
import { defineComponent } from 'vue';
export default defineComponent({
  name: 'SecondComp',
  components: {
    ThirdComp,
},
  data() {
    return {
    }
  }
})
</script>

<template>
    <h2>SecondComp</h2>
    <ThirdComp />
</template>

<style scoped>

</style>
