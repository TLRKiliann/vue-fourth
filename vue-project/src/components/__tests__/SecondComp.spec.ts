import { describe, it, expect } from 'vitest'

import { mount } from '@vue/test-utils'
import SecondComp from '../SecondComp.vue'

describe('SecondComp', () => {
  it('snapshot UI testing', () => {
    const wrapper = mount(SecondComp,{});
    expect(wrapper.text()).toMatchSnapshot()
  })
})