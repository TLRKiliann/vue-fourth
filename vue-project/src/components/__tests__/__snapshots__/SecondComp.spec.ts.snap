// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SecondComp > snapshot UI testing 1`] = `"SecondCompThirdComp"`;
